nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,7 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from typing import List, Sequence, Optional, Union
</pre><p>Please place Sequence after Optional, we want to add isort linting and this would fail</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>